### PR TITLE
Update front end event creation

### PR DESCRIPTION
**Pull request by keithbryant3**
Front end now takes inputs to determine how to repeat the event.

### CODE REVIEW

**Review by burdick-alex — APPROVED** [review]
Looks like a lot of files are the same. For the things that actually changed(like test_spec), they look good.

File: app/assets/javascripts/events_form.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  if(window.location.href.indexOf("edit") > -1 ) {
+    document.getElementById('switchButton').classList.toggle("hidden")
+    document.getElementById("instructions").innerHTML = ""
+
+    // Hide unnecessary information
+    event_repeatmonday_input= document.getElementById("event_repeatmonday_input")
+    event_repeatmonday= document.getElementById("event_repeatmonday")
+    event_repeatmonday_input.classList.toggle("hidden")
+	
+	event_repeattuesday_input= document.getElementById("event_repeattuesday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js:14]: kind of nit picking but can you fix the tabbing throughout this file

File: db/migrate/20210308025530_add_repeat_to_events.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class AddRepeatToEvents < ActiveRecord::Migration[6.0]
+  def change
+    add_column :events, :repeatmonday, :boolean
+	add_column :events, :repeattuesday, :boolean
```
Comment (varvind) [inline db/migrate/20210308025530_add_repeat_to_events.rb:4]: Same here tabbing is off

**Review by varvind — CHANGES_REQUESTED** [review]
Looks good for the most part just update the tabbing, also make sure to test heroku, but as far as I can see heroku looks good too

File: app/assets/javascripts/events_form.js (hunk 1 above)
Comment (keithbryant3) [reply app/assets/javascripts/events_form.js:14]: Done.

File: db/migrate/20210308025530_add_repeat_to_events.rb (hunk 2 above)
Comment (keithbryant3) [reply db/migrate/20210308025530_add_repeat_to_events.rb:4]: Done.

**Review by rhrussell — CHANGES_REQUESTED** [review]
There are only three changes that need to be done before I can incorporate your front-end code with my back-end code that I need.

For the file app/admin/events.rb:

Change permit_params by getting rid of all the repeat variables(days and weeks) because I believe that means they are required. Even if that is not the case, they are not variables that need to be part of the database nor event model. Other than that, that file is fine with the front-end.

And for the file db/migrate/20210308025530_add_repeat_to_events.rb

Change the rb file for events in the migrate folder back to its original state because these variables do not need to be in the database or part of an event. If this is added to the database, then every event will have variables saying if it repeats on that day or not which is not necessary.

And for the file db/schema.rb

Change this file (probably to its original state as well) so that it works with the migrate and does not add anything unnecessary to the database.

I have gone over these critiques with Arvind and he agrees that that is what needs to be done. But other than, great job man with all of the other stuff!

**Review by rhrussell — APPROVED** [review]
Everything looks good. Let you know if this works with my code

File: app/assets/javascripts/events_form.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeattuesday= document.getElementById("event_repeattuesday")
+    event_repeattuesday_input.classList.toggle("hidden")
+	
+	event_repeatwednesday_input= document.getElementById("event_repeatwednesday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: still some weird tabbing here

File: app/assets/javascripts/events_form.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeatwednesday= document.getElementById("event_repeatwednesday")
+    event_repeatwednesday_input.classList.toggle("hidden")
+	
+	event_repeatthursday_input= document.getElementById("event_repeatthursday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here too

File: app/assets/javascripts/events_form.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeatthursday= document.getElementById("event_repeatthursday")
+    event_repeatthursday_input.classList.toggle("hidden")
+	
+	event_repeatfriday_input= document.getElementById("event_repeatfriday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here

File: app/assets/javascripts/events_form.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeatfriday= document.getElementById("event_repeatfriday")
+    event_repeatfriday_input.classList.toggle("hidden")
+	
+	event_repeatsaturday_input= document.getElementById("event_repeatsaturday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here

File: app/assets/javascripts/events_form.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeatsaturday= document.getElementById("event_repeatsaturday")
+    event_repeatsaturday_input.classList.toggle("hidden")
+	
+	event_repeatsunday_input= document.getElementById("event_repeatsunday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here

File: app/assets/javascripts/events_form.js [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+    event_repeatsunday= document.getElementById("event_repeatsunday")
+    event_repeatsunday_input.classList.toggle("hidden")
+	
+	event_repeatweeks_input= document.getElementById("event_repeatweeks_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here

File: app/assets/javascripts/events_form.js [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+    document.getElementById("instructions").innerHTML = "What days should the event repeat for?"
+    i--
+  }
+
+  // toggles hidden for all elements, meaning Repeating information will be hidden
+    event_repeatmonday_input= document.getElementById("event_repeatmonday_input")
+    event_repeatmonday= document.getElementById("event_repeatmonday")
+    event_repeatmonday_input.classList.toggle("hidden")
+	
+	event_repeattuesday_input= document.getElementById("event_repeattuesday_input")
```
Comment (varvind) [inline app/assets/javascripts/events_form.js]: here and the rest on this file

**Review by varvind — CHANGES_REQUESTED** [review]
Still some tabbing issues on the JS file, fix those real quick and then we are good to go

**Pull request merged**